common/build/eslint-plugin-fluid/src/test/example/no-unchecked-record-access/indexedRecordOfStrings.ts

```diff
@@ -94,3 +94,40 @@ aImplicitType.length; // ok: aImplicitType is the continuation of the inferred t
 
 extendedIndexedRecordOfStrings.a.length; // ok: Accessing string property of extendedIndexedRecordOfStrings is allowed
 extendedIndexedRecordOfStrings.b.length; // defect: Accessing length of index property 'b', but 'b' might not be present
+
+interface NestedIndexSignatureType {
+	[FirstNestedKey: string]: {
+		[SecondNestedKey: string]: {
+			[ThirdNestedKey: string]: string;
+		};
+	};
+}
+
+function TestNullish(
```

Comment:
Nit: docs explaining the test case? 😁

Reply:
Also nit: this file has a lot of things going on. Assuming there are distinct test scenarios / cases, I wonder if it would be worth breaking this code up a bit to make the distinctions a bit more clear. You could do this with separate files, `// #region ` blocks, or just `{ }` scope blocks.

Probably not for this PR, just a thought for future maintenance 🙂